### PR TITLE
fix unexpected error in 'handling receipt'

### PR DESCRIPTION
**Pull request by lyfe00011**

### CODE REVIEW

File: src/WABinary/encode.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ export const encodeBinaryNode = (
 	const pushInt16 = (value: number) => {
 		pushBytes([(value >> 8) & 0xff, value & 0xff])
 	}
+
```
Comment (adiwajshing) [inline src/WABinary/encode.ts:28]: can remove this redundant change?

Comment (lyfe00011) [reply src/WABinary/encode.ts:28]: That from eslint fix

File: src/Socket/messages-recv.ts [hunk 2]
```diff
@@ -383,7 +383,7 @@ export const makeMessagesRecvSocket = (config: SocketConfig) => {
 							ids.map(id => ({
 								key: { ...key, id },
 								receipt: {
-									userJid: jidNormalizedUser(attrs.participant),
+									userJid: attrs.participant ? jidNormalizedUser(attrs.participant) : undefined,
```
Comment (adiwajshing) [inline src/Socket/messages-recv.ts:386]: I'm not sure how you're getting a group node without the participant? You've the log for that?

File: src/WABinary/encode.ts (hunk 1 above)
Comment (lyfe00011) [reply src/WABinary/encode.ts:28]: Btw i will do at evening

File: src/Socket/messages-recv.ts (hunk 2 above)
Comment (lyfe00011) [reply src/Socket/messages-recv.ts:386]: Yesh faced issue stop receiving messages and there is no participants

**Comment by ibnumardini** [comment]
i have same error: 

`TypeError: Cannot destructure property 'user' of '(0 , exports.jidDecode)(...)' as it is undefined.\n    at jidNormalizedUser (..../node_modules/@adiwajshing/baileys/lib/WABinary/jid-utils.js:49:13)\n    at ..../node_modules/@adiwajshing/baileys/lib/Socket/messages-recv.js:325:71\n    at Array.map (<anonymous>)\n    at ..../node_modules/@adiwajshing/baileys/lib/Socket/messages-recv.js:322:59\n    at ..../node_modules/@adiwajshing/baileys/lib/Utils/make-mutex.js:16:24\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at handleReceipt (..../node_modules/@adiwajshing/baileys/lib/Socket/messages-recv.js:312:9)`

**Pull request closed without merging**